### PR TITLE
Added information about OWIN environment

### PR DESCRIPTION
**Pull request by vanillajonathan**

### CODE REVIEW

File: aspnet/fundamentals/owin.rst [hunk 1]
```diff
@@ -101,6 +101,16 @@ This `sample  <https://github.com/aspnet/Docs/tree/master/aspnet/fundamentals/ow
 .. image:: owin/_static/websocket-test.png
 
 
+OWIN environment
+----------------
+You can construct a OWIN environment using the ``HttpContext``.
```
Comment (Rick-Anderson) [inline aspnet/fundamentals/owin.rst]: You can construct an OWIN environment object using ~~the~~ ``HttpContext``.

**Comment by Rick-Anderson** [comment]
@danroth27  please review

**Comment by Rick-Anderson** [comment]
@danroth27  we need a strategy  to get small changes migrated with minimal friction.

**Comment by danroth27** [comment]
:shipit:

**Pull request merged**